CHANGELOG.org

```diff
@@ -14,7 +14,9 @@
   * Change ~ruff-lsp~ to ~ruff~ for python lsp client. All ~ruff-lsp~ customizable variable change to ~ruff~. Lsp server command now is ~["ruff" "server"]~ instead of ~["ruff-lsp"]~.
   * Add futhark support
   * Optimize overlay creation by checking window visibility first
-
+  * Replace the per-interface ~(INTERFACE ...)~ pcase forms with a single,
+    unified ~(lsp-interface INTERFACE ...)~ form. The per-interface forms are no
+    longer generated. *This is a breaking change.* (See #4430.)
```

Comment:
I'm definitely open to suggestions on the CHANGELOG entry.

Reply:
Also, does the unreleased version stay 9.0.1, or should that go to 10.0.0 now?

Reply:
9.0.1 is good.